### PR TITLE
Improve tests and README

### PR DESCRIPTION
**Pull request by arichiardi**
This patch improves tests, bumps some dependency and improves a README.

### CODE REVIEW

File: src/task/rest_resources_viz/extractor.clj [hunk 1]
```diff
@@ -18,8 +17,7 @@
             [com.rpl.specter :as sp]
             [rest-resources-viz.spec :as rspec]))
 
-(defn instrument-all []
-  (run! stest/instrument (stest/instrumentable-syms)))
+(s/def ::non-empty-string (s/and string? (complement str/blank?)))
```
Comment (ElChache) [inline src/task/rest_resources_viz/extractor.clj]: not used

File: test/task/rest_resources_viz/extractor_test.clj [hunk 2]
```diff
@@ -1,17 +1,20 @@
...
+(run! stest/instrument (stest/instrumentable-syms))
 
 (def test-state (atom nil))
 
 (defn wrap-setup
   [f]
   (try
-    (swap! test-state assoc :graph-data (-> "graph-data.edn"
-                                            io/resource
+    (swap! test-state assoc :graph-data (-> (io/file "target" "graph-data.edn")
```
Comment (ElChache) [inline test/task/rest_resources_viz/extractor_test.clj]: it's in target? kindda odd...

**Review by ElChache — COMMENTED** [review]
just one oddity, and one not used thing

**Comment by arichiardi** [comment]
Thanks @ElChache addressed and merging!

**Pull request merged**